### ISSUE TITLE
Can phone volume buttons be hotkeys for the mobile browser when remote controlling shabad os on the same network?

### ISSUE THREAD

**Issue by bhajneet** [ShabadOS/desktop#417]

**Comment by Harjot1Singh** [comment]
I originally thought “no way”, but actually, there is a possibility, obviously only when the phone is unlocked: https://davidwalsh.name/javascript-volume

**Comment by bhajneet** [comment]
Does that also change the system volume?

Regardless, do we want to lock users out of using their volume button? I don't think we should hijack the user without an option. And I'd rather not create an option for this until we have a dedicated app.

**Comment by Harjot1Singh** [comment]
Yep, it’d also change their system volume, so no hijacking as far as I can tell right now

**Comment by bhajneet** [comment]
Let's wait to make it proper in mobile
